### DISCUSSION TITLE
Minimal installation to run ImageMagick from command line in Windows

### DISCUSSION THREAD

**Discussion by tedysh** [ImageMagick/ImageMagick#5784] [Consulting]
Is there a **minimal installation** (rather than the full install) that would allow running ImageMagick from the Windows command line, for the purpose of compressing and resizing a jpg image? 
Otherwise, is there a core file, or set of files that are required in order to interpret a command line (compress and resize jpg image only)?

**Comment by snibgo** [comment]
You could build a version of IM with only the image format coders that you need. It would still contain loads of code that you don't need.

If all you want is to read and write JPEG, with resizing, I suggest using specialist JPEG tools, such as cjpeg and djpeg.
